src/common/components/repository-row/index.js

```diff
@@ -21,12 +21,6 @@ const FILE_NAME_CLAIM_URL = 'https://myapps.developer.ubuntu.com/dev/click-apps/
 
 const LEARN_THE_BASICS_LINK = 'https://snapcraft.io/docs/build-snaps/your-first-snap';
 const INSTALL_IT_LINK = 'https://snapcraft.io/create/';
-const tickIcon = (
-  <img
```

Comment:
Seems like instead of removing it it could just be turned into:

```js
const tickIcon = (
   <span className={ `${styles.icon} ${styles.tickIcon}` } />
)
```

Which would simply work without the need to inline the `<span>` with class names everywhere.
Also, it would be easier for later to extract it as separate icon component that we might do.